docs/elasticsearch-spec.asciidoc

```diff
@@ -22,7 +22,7 @@ Before you deploy and run ECK, take some time to look at the basic and advanced
 - <<{p}-group-definitions>>
 - <<{p}-pod-disruption-budget>>
 - <<{p}-advanced-node-scheduling,Advanced Elasticsearch node scheduling>>
-
+- <<{p}-snapshot,Create automated snapshots>>
```

Comment:
Would it be more consistent if we don't use a verb? Just "Automated snapshots"? We have nominal sentences (I think that's the name:)) in the other titles (so "Virtual memory" instead of "Set virtual memory", "Custom HTTP certificate" instead of "Add custom HTTP certificate" etc.).